sdk/zksend/src/links.ts

```diff
@@ -33,24 +35,38 @@ const DEFAULT_ZK_SEND_LINK_OPTIONS = {
 	client: new SuiClient({ url: getFullnodeUrl('mainnet') }),
 };
 
-const SUI_COIN_TYPE = normalizeStructTag('0x2::coin::Coin<0x2::sui::SUI>');
+const SUI_COIN_TYPE = normalizeStructTag('0x2::sui::SUI');
```

Comment:
The SDK also has SUI_TYPE_ARG as an export I think

Reply:
unfortunately it's not normalized, but updated to use that